spec/spec_helper_acceptance.rb

```diff
@@ -42,6 +42,8 @@
       on host, puppet('module', 'install', 'puppet-rabbitmq'), { :acceptable_exit_codes => [0,1] }
       on host, puppet('module', 'install', 'fsalum-redis'), { :acceptable_exit_codes => [0,1] }
       on host, puppet('module', 'install', 'puppetlabs-apt'), { :acceptable_exit_codes => [0,1] }
+      on host, puppet('module', 'install', 'puppetlabs-dsc'), { :acceptable_exit_codes => [0,1] }
```

Comment:
acceptance testing is not currently happening, so this change should not be here. We're redoing all of this, so it would likely just cause a merge conflict.